pkg/operator/status.go

```diff
@@ -294,6 +295,13 @@ func (c *Controller) syncStatus(cr *imageregistryv1.Config, deploy *appsapi.Depl
 
 	updateCondition(cr, operatorapiv1.OperatorStatusTypeProgressing, operatorProgressing)
 
+	if isDeploymentStatusAvailable(deploy) {
```

Comment:
Can't we look at `operatorAvailable.Status` here, instead of calling out to `isDeploymentStatusAvailable` again?  And do we need `.deployUnavailableSince` when we should already have `LastTransitionTime` on the `Available` status (although maybe not on `operatorAvailable` until we use some tooling syncs it into the existing `Conditions` object).

Reply:
Yes, this makes total sense. I will try to adjust according to your input, thanks for that.

Reply:
Done, PTAL.